include/fmt/os.h

```diff
@@ -379,56 +379,61 @@ struct ostream_params {
 #  endif
 };
 
+class file_buffer final : public detail::buffer<char> {
```

Comment:
`detail` namespace qualification is not needed here because this is in the `detail` namespace.

Reply:
```suggestion
class file_buffer final : public buffer<char> {
```